common/version.h.in

```diff
@@ -0,0 +1,7 @@
+#define VERSION "@VERSION@"
+#define MPVCOPYRIGHT "Copyright © 2000-2023 mpv/MPlayer/mplayer2 projects"
+#ifndef NO_BUILD_TIMESTAMPS
+#define BUILDDATE __DATE__ " " __TIME__
```

Comment:
Wouldn't this cause problems wrt Reproducible Builds?
See (f.e.) https://reproducible-builds.org/docs/source-date-epoch/ for a way to prevent that issue

Reply:
You can disable the build date with `NO_BUILD_TIMESTAMPS` or set it to anything you want with `SOURCE_DATE_EPOCH` https://gcc.gnu.org/onlinedocs/cpp/Environment-Variables.html#index-SOURCE_005fDATE_005fEPOCH which compilers respect and set DATE/TIME accordingly.

Nothing has been changes compared to previous status quo. Actually only actual change in this PR is slightly different date string, everything else works the same way.

Reply:
Ok. Thought I'd mention it as I mostly see those macros getting replaced with something else.
But one can certainly work around that.